.github/workflows/docker_build.yml

```diff
@@ -14,10 +14,12 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
 
-      - name: Extract Git tag or branch slug
+      - name: Extract Git tag or fallback to 'dev'
         run: |
           if [[ "${GITHUB_REF}" == refs/tags/* ]]; then
             SLUG="${GITHUB_REF#refs/tags/}"
+          else
+            SLUG="dev"
```

Comment:
Why not `latest`?

Reply:
because this is also when you push after merge and can be overwritten all the time, latest will be created for the latest TAG with version 3.12.